debug_scripts/estimate_epoch_start_time.py

```diff
@@ -80,20 +82,35 @@ def get_exponential_weighted_epoch_lengths(url,
     return epoch_lengths, exponential_weighted_average_epoch_length
 
 
+# Function to check if timezone is valid
+def is_valid_timezone(timezone_str):
+    try:
+        pytz.timezone(timezone_str)
+        return True
+    except pytz.UnknownTimeZoneError:
+        return False
+
+
 # Function to approximate future epoch start dates
 def predict_future_epochs(starting_epoch_timestamp, avg_epoch_length,
```

Comment:
Add a comment here saying that the time in the returned `future_epochs` are in the tartget time zone.